recipes/stress-ng/build.sh

```diff
@@ -0,0 +1,4 @@
+
+make && make install
```

Comment:
make install into which directory? Are you sure you don't need to specify $PREFIX

Reply:
You can run make lite-test or make test here as well